devicetypes/smartenit/moisture-sensor-child.src/moisture-sensor-child.groovy

```diff
@@ -0,0 +1,23 @@
+/**
+ *  Moisture Sensor Child
+ *
+ *  Copyright 2021 Luis Contreras
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ */
+metadata {
+	definition (name: "Moisture Sensor Child", namespace: "Smartenit", author: "Luis Contreras") {
+		capability "Configuration"
+		capability "Refresh"
+		capability "Water Sensor"
+		capability "Health Check"
+		capability "Sensor"
+	}
```

Comment:
Please double-check if You need to add here any of those functions (that will call a parent dth code):
- `installed()`
- `updated()`
- `configure()`

Below implementations will be needed:
- `ping()` that will call `refresh()`
- `refresh()` that will call parent method -> for example: `parent.refreshChild()`

Reply:
Oh okay! Sorry for the delay, I just returned from vacation. In this case, I don't really need any of those implementations. The child device moisture state is obtained from the parent device's IAS Zone status request. The child state is embedded in the Alarm2 state and both states are updated at the same time.

Reply:
Sure, but somehow the refresh() function in the parent DTH has to be called when You trigger it in this child device, right ?